From 573d62152fb3bdbf803f6a65d3f996bf9b99ded1 Mon Sep 17 00:00:00 2001
Message-Id: <573d62152fb3bdbf803f6a65d3f996bf9b99ded1.1346940159.git.minovotn@redhat.com>
In-Reply-To: <c629acdc74c5b775c5cf7a7a3cb20ac833bcd0cc.1346940159.git.minovotn@redhat.com>
References: <c629acdc74c5b775c5cf7a7a3cb20ac833bcd0cc.1346940159.git.minovotn@redhat.com>
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 16 Aug 2012 11:39:17 +0200
Subject: [PATCH 15/18] usb storage: DPRINTF fixup

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1345117160-21046-16-git-send-email-kraxel@redhat.com>
Patchwork-id: 40920
O-Subject: [RHEL-6.4 qemu-kvm PATCH 15/18] usb storage: DPRINTF fixup
Bugzilla: 805172
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

---
 hw/usb-msd.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/usb-msd.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hw/usb-msd.c b/hw/usb-msd.c
index 7c5e87c..8c074a5 100644
--- a/hw/usb-msd.c
+++ b/hw/usb-msd.c
@@ -199,8 +199,8 @@ static void usb_msd_send_status(MSDState *s, USBPacket *p)
 {
     int len;
 
-    DPRINTF("Command status %d tag 0x%x, len %zd\n",
-            s->csw.status, le32_to_cpu(s->csw.tag), p->iov.size);
+    DPRINTF("Command status %d tag 0x%x, len %d\n",
+            s->csw.status, le32_to_cpu(s->csw.tag), p->len);
 
     assert(s->csw.sig == cpu_to_le32(0x53425355));
     len = MIN(sizeof(s->csw), p->len);
-- 
1.7.11.4

